api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -238,6 +243,34 @@ public void setUserService(UserService userService) {
     return ResponseEntity.ok(resp);
   }
 
+  @Override
+  @AuthorityRequired({Authority.MANAGE_CLUSTERS})
```

Comment:
Thanks – point #2 is probably the most convincing to me. I can get behind the idea that tying admin-style functions to a production release is the best way to guarantee that we don't accidentally run non-security-reviewed code against prod. Would be a net win for production safety as well as security.

Thinking about a couple follow-ups if we embrace that direction:

 - We may want to begin sanity-checking our authority grants on prod... e.g. ensure "ADMIN" or "DEVELOPER" authorities are (eventually) only ever attached to bona-fide RW devs or DRC team members. This could be a simple cron job, verifying that dev-authorized users have contact emails at verily.com, vumc.org, or broadinstitute.org (or some other heuristic).
 - Maybe we should find a way to *prevent* command-line tools from connecting to prod, or at least not without an audit-logged reason behind doing so (akin to break-glass acess). Not sure what this work would actually look like though... perhaps adding some extra protections to project.rb, and/or to the command-line tools Java classes?

Reply:
Yes, I think we can first establish a pattern, then perhaps abstract to a library if the code is non-trivial. Happy to file a ticket on the validation sanity check (something in set-authority might suffice) if the rest of this PR is accepted.